### PR TITLE
APP-3193 Selection & Awards page.

### PR DESCRIPTION
**Pull request by joanofdart**
Related to #3193

### CODE REVIEW

File: libs/movie/src/lib/movie/pages/reviews/reviews.component.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            <mat-error>Please enter a 4 digit date (e.g. 2020)</mat-error>
+            <mat-error *ngIf="control.hasError('required')">
+              Year is required.
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field appearance="outline" fxFlex>
+            <mat-label>Status</mat-label>
+            <mat-select matInput [formControl]="control.get('premiere')">
+              <mat-option *ngFor="let premiere of premiereType" [value]="premiere.slug">
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/reviews/reviews.component.html:32]: pls use static select here

File: libs/movie/src/lib/movie/pages/reviews/reviews.component.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+mat-form-field {
+  width: 100%;
+
+}
+
+h3 {
+  margin-bottom: 24px;
+}
+
+mat-divider {
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/reviews/reviews.component.scss:14]: it feels like the UI should stick to the design and not creating a new behavior on every page. Can you please check with tehm?

File: libs/movie/src/lib/movie/pages/reviews/reviews.component.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+@Component({
+  selector: 'movie-form-reviews',
+  templateUrl: './reviews.component.html',
+  styleUrls: ['./reviews.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MovieFormReviewscComponent {
+  form = this.shell.form;
+  public premiereType = staticModels.PREMIERE_TYPE;
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/reviews/reviews.component.ts:13]: you can remove that and use the static select. Input scope is just the string that says use `PREMIERE_TYPE`

File: libs/movie/src/lib/movie/pages/reviews/reviews.component.scss (hunk 2 above)
Comment (joanofdart) [reply libs/movie/src/lib/movie/pages/reviews/reviews.component.scss:14]: this is the same behavior from all the pages we've done so far.

File: libs/movie/src/lib/movie/pages/reviews/reviews.module.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+import { MovieFormReviewscComponent } from './reviews.component';
+
+@NgModule({
+  declarations: [MovieFormReviewscComponent],
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+
+    // Other Modules
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/reviews/reviews.module.ts:27]: nice!

**Pull request merged**